### PR TITLE
Added neo-scan-api container to CLI

### PR DESCRIPTION
**Pull request by nunojusto**
## Problem

Add neo-scan-api to CLI
Solving issue https://github.com/CityOfZion/neo-local/issues/71

## Solution

Adding neo-scan-sync

## Checklist

- [x] I have ran the tests locally.
- [x] I branched off of the `develop` branch and not `master`.
- [x] I did not change the `VERSION` file.

### CODE REVIEW

File: cli/stack/neo-scan-api.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	"github.com/docker/go-connections/nat"
+)
+
+// NewNeoScanSync creates a new service for the egistry.gitlab.com/cityofzion/neo-scan/sync:latest image.
+func NewNeoScanAPI() Service {
+	return Service{
+		Author: "cityofzion",
+		ContainerConfig: &container.Config{
+			Env: []string{
+				"NEO_NOTIFICATIONS_SERVER='http://notifications-server:8080/v1'",
```
[the remark this answers is not in the document]
Comment (nunojusto) [reply cli/stack/neo-scan-api.go]: done

File: cli/stack/neo-scan-api.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+				"DB_DATABASE=neoscan_prodv",
+				"REPLACE_OS_VARS=true",
+			},
+			ExposedPorts: map[nat.Port]struct{}{
+				"4000/tcp": {},
+			},
+		},
+		DependsOn: []string{"postgres", "neo-privatenet", "notifications-server"},
+		HostConfig: &container.HostConfig{
+			Links: []string{"neo-privatenet:30333", "neo-privatenet:30334", "neo-privatenet:30335", "neo-privatenet:30336"},
```
[the remark this answers is not in the document]
Comment (nunojusto) [reply cli/stack/neo-scan-api.go]: done

File: cli/stack/neo-scan-api.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+				"DB_USERNAME=postgres",
+				"DB_PASSWORD=postgres",
+				"DB_DATABASE=neoscan_prodv",
+				"REPLACE_OS_VARS=true",
+			},
+			ExposedPorts: map[nat.Port]struct{}{
+				"4000/tcp": {},
+			},
+		},
+		DependsOn: []string{"postgres", "neo-privatenet", "notifications-server"},
```
[the remark this answers is not in the document]
Comment (nunojusto) [reply cli/stack/neo-scan-api.go]: done

File: cli/stack/neo-scan-api.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+				"PORT=4000",
+				"REPLACE_OS_VARS=true",
+			},
+			ExposedPorts: map[nat.Port]struct{}{
+				"4000/tcp": {},
+			},
+		},
+		DependsOn: []string{
+			"postgres",
+			"neo-privatenet",
```
Comment (revett) [inline cli/stack/neo-scan-api.go]: A-Z needed here as well

Comment (nunojusto) [reply cli/stack/neo-scan-api.go]: done

File: cli/stack/neo-scan-api.go [hunk 5]
```diff
@@ -0,0 +1,53 @@
+package stack
+
+import (
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+)
+
+// NewNeoScanAPI creates a new service for the egistry.gitlab.com/cityofzion/neo-scan/sync:latest image.
```
Comment (revett) [inline cli/stack/neo-scan-api.go]:
```suggestion
// NewNeoScanAPI creates a new service for the 
// registry.gitlab.com/cityofzion/neo-scan/sync:latest image.
```

**Pull request merged**